mne/beamformer/_compute_beamformer.py

```diff
@@ -276,10 +276,10 @@ def _compute_beamformer(method, G, Cm, reg, n_orient, weight_norm,
             else:
                 rank_Cm = estimate_rank(Cm, tol='auto', norm=False,
                                         return_singular=False)
-                noise = noise[len(noise) - rank_Cm]
+            noise = noise[rank_Cm]
```

Comment:
I do not think this is correct. Note that we use ``linalg.eigh`` which returns the eigenvalues in ascending order. The old version gives the same results as Fieldtrip.

Reply:
aah, ascending order... how weird!